syntax = "proto3";
//python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. p2pseed.proto
service P2pSeed {
  rpc Hello(HelloRequest) returns (HelloResponse) {}
  rpc FindOnlineNode(FindOnlineNodeRequest) returns(FindOnlineNodeResponse){}

}
// node
message FindOnlineNodeRequest {
  string message = 1;
}

message FindOnlineNodeResponse {
  repeated int32 port = 1;
}

message HelloRequest {
  string message = 1;
  int32 port = 2;
}

message HelloResponse {
  string message = 1;
  repeated int32 port = 2;
}
